ckanfunctionaltests/api/conftest.py

```diff
@@ -29,6 +33,23 @@ def random_org_slug(base_url, rsession):
 
 @pytest.fixture()
 def random_pkg_slug(base_url, rsession):
-    response = rsession.get(f"{base_url}/action/package_list")
+    # make do with one of the first 200 because the full list is big & slow
+    response = rsession.get(f"{base_url}/action/package_list?limit=200")
     assert response.status_code == 200
-    return _random.choice(response.json()["result"])
+
+    for _ in range(8):
```

Comment:
Can the 8 be moved to a constant, this should make the code a little bit clearer.

Reply:
Actually I think I'll rewrite this whole block to be less silly and hideous entirely :)